documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -1,5 +1,11 @@
 # Current Release notes
 
+## Version 8.10.0
+
+### Resolved issues
+
+* (IDETECT-3776) Resolved misclassification issues for certain components in the `go.mod` file by improving identification of direct and indirect dependencies.
```

Comment:
We may want to add in the exact issue that was resolved.
re: "_recognize components in the go.mod file with "// indirect" as transitive dependencies_".
Suggested edit:
* (IDETECT-3776) Resolved an issue with not detecting certain components in `go.mod` files as transitive dependencies when marked with `// indirect`, by improving identification of direct and indirect dependencies.